zipkin/src/tracer.cc

```diff
@@ -108,7 +109,26 @@ void Tracer::reportSpan(Span &&span) {
   }
 }
 
-void Tracer::setReporter(ReporterPtr reporter) {
-  reporter_ = std::move(reporter);
+uint64_t Tracer::droppedSpanCount() const {
+  return reporter_->droppedSpanCount();
 }
+
+uint64_t Tracer::getAndResetDroppedSpanCount() {
+  return reporter_->getAndResetDroppedSpanCount();
```

Comment:
What would you think of adding a `reporter` accessor method to the Tracer class? Then you can get rid of all of these forwarding methods.

Reply:
I think that makes sense.